tests/gpflow/optimizers/test_natural_gradient.py

```diff
@@ -12,19 +12,23 @@
 class Setup:
```

Comment:
This is a bit out of place in the refactor / reorganization, but in the spirit of "tidy as you go", I think it stays.

Reply:
yeah, part of this was not wanting to have to think too much about how to consistently get tensorflow random numbers - I know there's tf.random.set_seed() but I trusted a RandomState object more...